flake.nix

```diff
@@ -7,17 +9,22 @@
 
     flake-parts.url = "github:hercules-ci/flake-parts";
     nixos-flake.url = "github:srid/nixos-flake";
+    systems.url = "github:nix-systems/default";
   };
 
-  outputs = inputs@{ self, ... }:
+  outputs = inputs:
     inputs.flake-parts.lib.mkFlake { inherit inputs; } {
-      systems = [ "x86_64-linux" "aarch64-darwin" "x86_64-darwin" ];
+      # systems = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

Comment:
This comment could be replaced with a link to [nix-systems/default/default.nix](https://github.com/nix-systems/default/blob/da67096a3b9bf56a91d16901293e51ba5b49a27e/default.nix).